cmd/kogito/command/project/delete_project_test.go

```diff
@@ -27,6 +27,8 @@ import (
 )
 
 func Test_DeleteProjectCmd_WhenWeSuccessfullyDelete(t *testing.T) {
+	teardown := test.OverrideKubeConfigAndCreateDefaultContext()
+	defer teardown()
```

Comment:
didn't understand why use defer here, does the OverrideKUbe... implements some close method?

Reply:
See the result of the function, we revert the changes in the local machine to not ruin with the dev environment.